README.md

```diff
@@ -1 +1,63 @@
-# Drone-Simulation
\ No newline at end of file
+# Drone-Simulation
+## Abstract
+
+A simulation of a basic drone model take-off up to a given altitude in a gazebo environment; with a PX4 autopilot supported by Mavros in Ubuntu.
+
+![Final Product](https://github.com/abyasingh/Drone-Simulation/blob/main/Images%20and%20Videos/Simultion%20of%20drone.png)
```

Comment:
images should be enclosed in ```<p></p>```  tags, so that they can be aligned in the center i.e ```align="center```